core/templates/pages/exploration-editor-page/translation-tab/services/translation-tab-active-mode.service.ts

```diff
@@ -21,17 +21,15 @@ require(
   'pages/exploration-editor-page/exploration-editor-page.constants.ajs.ts');
 
 angular.module('oppia').factory('TranslationTabActiveModeService', [
-  '$rootScope', 'TRANSLATION_MODE', 'VOICEOVER_MODE',
-  function($rootScope, TRANSLATION_MODE, VOICEOVER_MODE) {
+  'TRANSLATION_MODE', 'VOICEOVER_MODE',
+  function(TRANSLATION_MODE, VOICEOVER_MODE) {
     var activeMode = null;
     return {
       activateVoiceoverMode: function() {
         activeMode = VOICEOVER_MODE;
-        $rootScope.$broadcast('translationTabModeChange');
```

Comment:
How did yo figured out these are not required?

Because just have broadcast and we do not have on for these?

Reply:
Yes!